src/services/validator.js

```diff
@@ -1,10 +1,35 @@
 import Promise from 'bluebird';
 import yamlLint from 'yaml-lint';
-import { PensieveLinterError } from './errors';
+import jsyaml from 'js-yaml';
+import _ from 'lodash';
+import moment from 'moment';
```

Comment:
Moment adds `200kb` to you're bundle if you don't ignore the locales stuff, unfortunately, but we'd also prefer not to eject from create-react-app so I'd suggest using another date time library, or we could use it add PR the fix mentioned here: https://github.com/facebookincubator/create-react-app/issues/1060